frontend/static/locales/ru.json

```diff
@@ -10,10 +10,10 @@
   "Existing orders match yours!": "Существующие ордеры совпадают с вашими!",
```

Comment:
Didn't touch this file but pre-commit insisted on touching it

Reply:
Yes, I expected this could happen once we started commiting `/locales` changes into `v0.6.0`. The `ru.json` in Main branch was updated via contribution, it was then merged into the v0.6.0 branch. So the pre-commit script possibly found phrases in v0.6.0 that were not there in the updated `ru.json` . It is safe to add this `ru.json` reformatting!